build.sh

```diff
@@ -13,16 +13,13 @@ fi
 
 if [ "$VERSION" == "jessie" ]
 then
-  VERSION_NUMBER=8
+  export VERSION_NUMBER=8
 else
   echo "Invalid version $VERSION"
   exit 1
 fi
 
 cp -R third_party/docker/mkimage* mkdebootstrap/
 
-sed -i "s|%REPO%|$REPO|g" cloudbuild.yaml
-sed -i "s|%TAG%|$TAG|g" cloudbuild.yaml
-sed -i "s|%VERSION%|$VERSION|g" cloudbuild.yaml
-sed -i "s|%NUM%|$VERSION_NUMBER|g" cloudbuild.yaml
-$GCLOUD_CMD alpha container builds create . --config=cloudbuild.yaml --verbosity=info --gcs-source-staging-dir=gs://$BUCKET/staging --gcs-log-dir=gs://$BUCKET/logs
+envsubst '${REPO} ${TAG} ${VERSION} ${VERSION_NUMBER}' <cloudbuild.yaml >cloudbuild.yaml.out
```

Comment:
Note that we could skip the temp file by piping this into gcloud over stdin, and passing /dev/fd/0 in as the --config parameter.